projects/angular/src/forms/common/wrapped-control.spec.ts

```diff
@@ -415,6 +415,17 @@ export default function (): void {
         expect(this.input.getAttribute('aria-describedby')).toBe(null);
       });
 
+      it('adds the aria-describedby with helper and error ids', fakeAsync(function (this: TestContext) {
+        setupTest(this, WithControlAndError, TestControl3);
+        this.input.focus();
+        this.input.blur();
+        this.fixture.detectChanges();
+        tick();
+        const describedByIds: string[] = this.input.getAttribute('aria-describedby').split(' ');
+        const expectedIds = [`${this.input.id}-helper`, `${this.input.id}-error`];
+        expect(describedByIds).toEqual(expectedIds);
```

Comment:
```suggestion

        expect(this.input.getAttribute('aria-describedby')).toEqual(`${this.input.id}-helper ${this.input.id}-error`);
```

Same suggestion and add missing blank line for consistency.